packages/tenant-process/test/tenant.integration.test.ts

```diff
@@ -828,6 +832,200 @@ describe("Integration tests", () => {
+          await readLastEventByStreamId(
+            tenantWithCertifiedAttribute.id,
+            eventStoreSchema.tenant,
+            postgresDB
+          );
+
+        if (!writtenEvent) {
+          fail("Update failed: tenant not found in event-store");
+        }
+        expect(writtenEvent).toMatchObject({
+          stream_id: tenantWithCertifiedAttribute.id,
+          version: "1",
+          type: "TenantCertifiedAttributeRevoked",
+        });
+        const writtenPayload = protobufDecoder(
+          TenantCertifiedAttributeRevokedV2
+        ).parse(writtenEvent?.data);
+
+        const updatedTenant: Tenant = {
+          ...tenantWithCertifiedAttribute,
+          attributes: [
+            {
+              id: attributeId,
+              type: "PersistentCertifiedAttribute",
+              assignmentTimestamp: new Date(
+                Number(
+                  (
+                    writtenPayload.tenant!.attributes[0].sealedValue as {
+                      oneofKind: "certifiedAttribute";
+                      certifiedAttribute: TenantCertifiedAttributeV2;
+                    }
+                  ).certifiedAttribute.assignmentTimestamp
+                )
+              ),
+              revocationTimestamp: new Date(
+                Number(
+                  (
+                    writtenPayload.tenant!.attributes[0].sealedValue as {
+                      oneofKind: "certifiedAttribute";
+                      certifiedAttribute: TenantCertifiedAttributeV2;
+                    }
+                  ).certifiedAttribute.revocationTimestamp
+                )
+              ),
+            },
+          ],
+          kind: fromTenantKindV2(writtenPayload.tenant!.kind!),
+          updatedAt: new Date(Number(writtenPayload.tenant?.updatedAt)),
+        };
+        expect(writtenPayload.tenant).toEqual(toTenantV2(updatedTenant));
+      });
+      it("Should throw tenant not found", async () => {
+        await addOneAttribute(attribute, attributes);
+        expect(
+          tenantService.revokeCertifiedAttributeById(
+            targetTenant.id,
+            attributeId,
+            mockAuthData,
+            correlationId
+          )
+        ).rejects.toThrowError(tenantNotFound(requesterTenant.id));
+      });
+      it("Should throw attribute not found", async () => {
+        await addOneTenant(targetTenant, postgresDB, tenants);
+        await addOneTenant(requesterTenant, postgresDB, tenants);
+
+        expect(
+          tenantService.revokeCertifiedAttributeById(
+            targetTenant.id,
+            attributeId,
+            mockAuthData,
+            correlationId
+          )
+        ).rejects.toThrowError(attributeNotFound(attribute.id));
+      });
+      it("Should throw tenant is not a certifier", async () => {
+        const requesterTenant: Tenant = {
+          ...mockTenant,
+          updatedAt: currentDate,
+          name: "A requesterTenant",
+        };
```

Comment:
Can't you just use `mockTenant` here? `Name` and `updatedAt` aren't useful to thist test case